tasks/agent.py

```diff
@@ -162,7 +162,7 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
     args = {
         "go_mod": go_mod,
         "race_opt": "-race" if race else "",
-        "build_type": "-a" if rebuild else ("-i" if precompile_only else ""),
+        "build_type": "-a" if rebuild else "",
```

Comment:
Are these really related to the key changes in this PR to be included?

Reply:
Not directly indeed, I should do a separate PR